point_viewer_grpc_proto_rust/src/proto.proto

```diff
@@ -51,3 +53,17 @@ message GetPointsInBoxRequest {
 message GetPointsInBoxReply {
   repeated point_viewer.proto.Vector3f points = 1;
 }
+
+message GetPointsInFrustumRequest {
+  point_viewer.proto.Vector3f view_position = 1;
+  point_viewer.proto.Vector3f view_direction = 2;
+  point_viewer.proto.Vector3f view_up = 3;
```

Comment:
What does this parameter mean?
And the next one, fov_y, in radians?

Reply:
@aliksey 
view_up is the direction of up vector
fov_y is in degrees so it would be degrees*(pi/180) radians

Reply:
Yes, thank you

Reply:
The unit of fov_y is radians. Please rename the variable to fov_y_radians.